herd/AArch64Sem.ml

```diff
@@ -1244,7 +1254,14 @@ module Make
             movi_v r k shift ii
         | I_MOVI_S(var,r,k) ->
             movi_s var r k ii
-
+        | I_EOR_SIMD(r1,r2,r3) ->
+            let size = neon_sz r1 in
+            simd_op EOR size r1 r2 r3 ii
+        | I_ADD_SIMD(r1,r2,r3) ->
+            let size = neon_sz r1 in
+            simd_op ADD size r1 r2 r3 ii
+        | I_ADD_SIMD_S(r1,r2,r3) ->
+            simd_op ADD MachSize.Quad r1 r2 r3 ii
```

Comment:
I do not know if  I_EOR_SIMD_S makes sense. If so it would not be very difficult to implement it.

Reply:
Looking at the Architecture Reference Manual, there doesn't seem to be a scalar variant for the SIMD [Exclusive OR instruction](https://developer.arm.com/documentation/ddi0596/2020-12/SIMD-FP-Instructions/EOR--vector---Bitwise-Exclusive-OR--vector--); there is, however, a scalar variant for the SIMD [ADD instruction](https://developer.arm.com/documentation/ddi0596/2020-12/SIMD-FP-Instructions/ADD--vector---Add--vector--).